prime-angular-frontend/src/app/modules/adjudication/shared/components/health-authority-table/health-authority-table.component.html

```diff
@@ -28,7 +28,7 @@
             scope="col"> Organization Name
         </th>
         <td mat-cell
-            *matCellDef="let row;"> {{ row.name | default }} </td>
+            *matCellDef="let row;"> {{ isHealthAuthorityObject(row) ? row.name : "" }} </td>
```

Comment:
Do we not need the **default** filter anymore?

Reply:
If a row is a Health Authority, I display its name.  But if it's a Site Registration, I display blank.  The `default` filter as was used here put a  "-" if the row didn't have a name ... why should the `default` filter continue to be used?

Reply:
Got it. Didn't check the filter, just asked as thought it had more stuff to it and to make sure formatting remains consistent.